app/main.py

```diff
@@ -1,8 +1,32 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for pers in people:
+        person = Person(pers["name"], pers["age"])
+
+        if "wife" in pers:
+            wife_name = pers["wife"]
+
+            if wife_name in Person.people:
+                person.wife = Person.people[wife_name]
+                person.wife.husband = person
+
+        if "husband" in pers:
+            husband_name = pers["husband"]
+
+            if husband_name in Person.people:
+                person.husband = Person.people[husband_name]
+                person.husband.wife = person
```

Comment:
could u create wife_name\husband_name vars before ur if condition using get method, so u could use only one if condition instead of nested conditions?